mlir/include/mlir/IR/EnumAttr.td

```diff
@@ -168,6 +168,7 @@ class EnumAttrInfo<
 
   // Generate specialized Attribute class
   bit genSpecializedAttr = 1;
+  bit genParser = 1;
```

Comment:
Is it just parser or parser and printer?

Reply:
Ah duh - I was killing both by putting the guard where I did but I do intend to only kill `FieldParser`. Let me fix.